frontend/src/app/components/enterprise/enterprise-trial-waiting/ee-trial-waiting.component.sass

```diff
@@ -1,7 +1,7 @@
-#resend-link
+#op-ee-trial-waiting-resend-link
   float: right
 
-.status--confirmed
+.op-ee-trial-waiting-status--confirmed
   color: var(--button--alt-highlight-background-color)
-.status--waiting
+.op-ee-trial-waiting-status--waiting
```

Comment:
The base block class should be in this file (`.op-ee-trial-waiting`), and the status and resend stuff should be elements for that. If possible, always use the following style:

```
// file is called block.sass or block.component.sass
.op-block
  &--element
  &--another-element
```